temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -267,6 +267,86 @@ message GetRegionResponse {
     temporal.api.cloud.region.v1.Region region = 1;
 }
 
+
+message GetApiKeysRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100.
+    int32 page_size = 1;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+    // Filter api keys by owner id - optional.
+    string owner_id = 3;
+    // Filter api keys by owner type - optional.
+    // Possible values: user, service-account
+    string owner_type = 4;
+}
+
+message GetApiKeysResponse {
+    // The list of API keys in ascending ids order
+    repeated temporal.api.cloud.identity.v1.ApiKey api_keys = 1;
+    // The next page's token
+    string next_page_token = 2;
+}
+
+message GetApiKeyRequest {
+    // The id of the API key to get
+    string key_id = 1;
+}
+
+message GetApiKeyResponse {
+    // The API key
+    temporal.api.cloud.identity.v1.ApiKey api_key = 1;
+}
+
+message CreateApiKeyRequest {
+    // The spec for the API key to invite
+    // Create api key only supports service-account owner type
+    temporal.api.cloud.identity.v1.ApiKeySpec spec = 1;
+    // The id to use for this async operation - optional
+    string async_operation_id = 2;
+}
+
+message CreateApiKeyResponse {
+    // The id of the API Key created
+    string key_id = 1;
+    // The token of the API Key created
```

Comment:
if this is sensitive/secret, should we prefix this or indicate that on the comment somewhere?

Reply:
Added a comment.